src/ds/logger.h

```diff
@@ -295,12 +305,18 @@ namespace logger
     size_t line_number;
     std::string ll_str;
     std::string msg;
+    uint16_t thread_id;
 
   public:
     LogLine(Level ll, const char* file_name, size_t line_number) :
       file_name(file_name),
       line_number(line_number),
-      log_level(ll)
+      log_level(ll),
+#ifdef INSIDE_ENCLAVE
+      thread_id(thread_ids[std::this_thread::get_id()])
+#else
+      thread_id(999)
```

Comment:
As discussed, I'm not super keen on this value. What about we print "`--`" or something non-numerical for the host instead?

Reply:
I think we want to log from various host threads with their own host thread ids, potentially. Can we have negative host ids and positive enclave ids? Or perhaps enclave from 0 and host from 100?